tests/series/test_hash.py

```diff
@@ -176,6 +176,13 @@ def test_murmur3_32_hash_date():
     assert hashes.to_pylist() == [-653330422, None]
 
 
+def test_murmur3_32_hash_time():
+    arr = Series.from_pylist([time(22, 31, 8), None])
+    assert arr.datatype() == DataType.time("us")
+    hashes = arr.murmur3_32()
+    assert hashes.to_pylist() == [-662762989, None]
```

Comment:
Similar to the timestamp tests below, we should confirm that this hash is the same when we use the nanoseconds resolution.